cmd/config/internal/commands/source.go

```diff
@@ -71,7 +71,7 @@ func (r *SourceRunner) runE(c *cobra.Command, args []string) error {
 
 	var inputs []kio.Reader
 	for _, a := range args {
-		inputs = append(inputs, kio.LocalPackageReader{PackagePath: a})
```

Comment:
Do we need to update the run command as well?  We probably want `source DIR | run`and `run DIR` to use the same file matching.

Reply:
@pwittrock Does `fn run` depend on the file path annotation  ? As long as it doesn't depend , it should not have an effect IMO. The resource list is in yaml format only.

Reply:
Would you add a test to confirm that `run DIR/` and `source | run ` produce the same result when there are json inputs?

Reply:
Planning to add an e2e test something like 

`kpt fn source myDir1/ | kpt fn run --image gcr.io/kpt-functions/label-namespace -- label_name=color label_value=orange | kpt fn sink myDir2/`

But it would need to pull an image. Is that fine and if so what is the correct place to do it. I tested manually and it works with json files.

Reply:
@pwittrock We should have e2e tests something like (do we already have such explicit e2e test for functions involving source, run and sink)

`config fn source /myDir1 | config fn run --image gcr.io/kpt-functions/label-namespace -- label_name=color label_value=orange | config fn sink /myDir2`

which involves pulling down an image. Let me know if this is fine so I can add. I tested manually on local with json files and it works fine.

Reply:
Yes -- but the test doesn't pull down an image -- it builds the image and uses the local copy

https://github.com/kubernetes-sigs/kustomize/blob/master/cmd/config/internal/commands/e2e/e2e_test.go#L747

Adding a test to that package SGTM

Reply:
@pwittrock added a test.